ChatBot/chatbot.py

```diff
@@ -0,0 +1,63 @@
+print("Hello! My name is Gaben_bot.")
+
+print("I was created in 2022.")
+
+
+
+print("Please, remind me your name.")
```

Comment:
print("Please, remind me your name.")
user_input = input(">")
print("What a great name you have,", user_input,'!')